app-header/app-header.html

```diff
@@ -0,0 +1,223 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../../iron-flex-layout/iron-flex-layout.html">
+<link rel="import" href="../scroll-behaviors/custom-scroller-behavior.html">
+
+<!--
+`app-header` can add transitions to the elements such as app-toolbar based on the scroll position.
+
+Example:
+
+Add a drawer in the left side of the screen:
+
+    <app-header condenses>
+      <app-toolbar>
+        <div title>App name</div>
+      </app-toolbar>
+    </app-drawer>
+
+@demo demo.html
+-->
+
+<dom-module id="app-header">
+
+  <style>
+
+    :host {
+      display: block;
+    }
+
+    :host::after {
+      content: '';
+      position: absolute;
+      left: 0px;
+      right: 0px;
+      bottom: -5px;
+      width: 100%;
+      height: 5px;
+      pointer-events: none;
+      opacity: 0;
+      box-shadow: inset 0px 5px 6px -3px rgba(0, 0, 0, 0.4);
+      -webkit-transition: opacity 0.4s;
+      transition: opacity 0.4s;
```

Comment:
we can remove `-webkit-transition` since `transition` is supported in all the browsers that Polymer supports.